docs/src/main/tut/docs/cassandra/README.md

```diff
@@ -4,6 +4,265 @@ title: Cassandra
 permalink: /docs/cassandra
 ---
 
-# Cassandra
 
-WIP
\ No newline at end of file
+# Freestyle-Cassandra
+
+[Cassandra] atop **Freestyle** is **`frees-cassandra`**.
+
+Freestyle Cassandra is a purely functional library for interacting with Cassandra supporting both,
+[LowQueryAPI] and [StringQueryInterpolation].
+
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+  - [What’s frees-cassandra](#whats-frees-cassandra)
+  - [Installation](#installation)
+  - [About Freestyle Cassandra](#about-freestyle-cassandra)
+  - [Freatures](#freatures)
+    - [Low level API](#low-level-api)
+    - [String Interpolator](#string-interpolator)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## What’s frees-cassandra
+
+[frees-cassandra] is a library to interact with cassandra built atop Free and following the
+[Freestyle] philosophy, being macro-powered. We use Troy under the scenes for modeling purposes 
+and query schema validation, as well as Datastax Cassandra Driver for connecting to a Cassandra 
+Cluster.
+
+## Installation
+
+Add the following resolver and library dependency to your project's build file. 
+
+For Scala `2.11.x` and `2.12.x`:
+
+```scala
+Resolver.bintrayRepo("tabdulradi", "maven")
+libraryDependencies += "io.frees" %% "frees-cassandra" % "0.0.4" 
+```
+
+## About Freestyle Cassandra
+
+> [Troy](https://github.com/schemasafe/troy) is a type-safe & compile-time-checked wrapper around 
+> the Cassandra driver that allows you to write raw CQL queries, validating them against your 
+> schema, and showing errors at compile-time. 
+ 
+Freestyle-Cassandra provides 2 different APIS, a low level one, letting you define a query and bind 
+values to it, and a String Interpolator one, which allows us to write raw queries and validate them 
+at compile time against a previously defined schema.
+
+In the upcoming sections, we'll take a look at both features and how we can take advantage of them.
+
+## Public APIs
+*Frees-Cassandra* provides a set of algebras to interact with the different pieces of a Cassandra 
```

Comment:
Maybe its worth to link `algebras` to the freestyle docs (http://frees.io/docs/core/algebras/)

Reply:
👍